tgui/packages/tgui/interfaces/Photocopier.js

```diff
@@ -95,6 +95,17 @@ const Actions = (props, context) => {
         content="Scan"
         onClick={() => act('scandocument')}
       />
+      {!!data.issilicon && (
```

Comment:
At the top, just below `const { act, data }`, you should put `const { issilicon } = data;` then just call that instead of data.issilicon. Then, can you also try removing the double `!!`, its often unneeded.

Reply:
Tried removing the double `!!` but doing so resulted in the following 
![image](https://github.com/ParadiseSS13/Paradise/assets/19361017/53a0aa71-a6ce-4560-9fd5-ecda6ad18715)

Other requested changes have been addressed.

Reply:
The image was what a human sees when they look at the photocopier when `!!` is removed. With `!!` the 0 is gone.
![image](https://github.com/ParadiseSS13/Paradise/assets/19361017/ba284193-1c35-45cd-8af6-7e3eaf7976d4)